src/Employer/Employer.Web/Views/Part1/Wage/Wage.cshtml

```diff
@@ -36,7 +36,7 @@
             <div class="form-group">
                 <label asp-for="WeeklyHours" class="form-label-bold">How many hours will the apprentice work each week, including training?</label>
                 <span asp-validation-for="WeeklyHours" class="error-message"></span>
-                <input asp-for="WeeklyHours" class="form-control form-control-1-8" type="number" step="0.01">
+                <input asp-for="WeeklyHours" class="form-control form-control-1-8" type="number" step=".25" min="16" max="48">

```

Comment:
Is the UI getting too clever here? Should the decision about the hours be done as business rule?  If we can put these limitations in the HTML then maybe we should update the supporting caption to provide the allowable range.

Reply:
Again, this is just a light way of restricting values if the user opts to use the up/down arrows. In terms of the 16 and 48, yes you are right, they need to be reflective of the business rule we will validate on server side and maybe we are more informative in the label/caption beforehand.

Reply:
👍